README.md

```diff
@@ -40,85 +40,194 @@ The project uses [Poetry](https://python-poetry.org/) to build the GQLAlchemy Py
+<br>
+<br>
+
+```python
+from gqlalchemy import Memgraph, Node, Relationship, Field
+from typing import Optional
+
+db = Memgraph()
+
+class User(Node, index=True, db=db):
+    id: str = Field(index=True, exist=True, unique=True, db=db)
+
+class Language(Node):
+    name: str = Field(unique=True, db=db)
+
+class Speaks(Relationship, type="SPEAKS"):
+    pass
+
+user = User(id="3", username="John").save(db)
+language = Language(name="en").save(db)
+speaks_rel = Speaks(
+    _start_node_id = user._id,
+    _end_node_id = language._id
+).save(db)
+
+loaded_user = User(id="3").load(db=db)
+print(loaded_user)
+loaded_speaks = Speaks(
+        _start_node_id=user._id,
+        _end_node_id=language._id
+    ).load(db)
+print(loaded_speaks)
+```
+</details>
+
+<details>
+<summary>Query builder 🔨</summary>
+<br>
+When building a Cypher query, you can use a set of methods that are wrappers around Cypher clauses. 
+<br>
+<br>
 
 ```python
-from gqlalchemy import Memgraph
-
-memgraph = Memgraph("127.0.0.1", 7687)
-memgraph.execute("CREATE (:Node)-[:Connection]->(:Node)")
-results = memgraph.execute_and_fetch("""
-    MATCH (from:Node)-[:Connection]->(to:Node)
-    RETURN from, to;
-""")
-
-for result in results:
-    print(result['from'])
-    print(result['to'])
+from gqlalchemy import create, match
+
+query_create = create()
+        .node(labels="Person", name="Leslie")
+        .to(edge_label="FRIENDS_WITH")
+        .node(labels="Person", name="Ron")
+        .execute()
+
+query_match = match()
+        .node(labels="Person", variable="p1")
+        .to()
+        .node(labels="Person", variable="p2")
+        .where(item="p1.name", operator="=", literal="Leslie")
+        .return_({"p1":"p1"})
+        .execute()
 ```
+</details>
 
-## Query builder example
+<details>
+<summary>Manage streams 🚰</summary>
+<br>
 
-As we can see, the example above can be error-prone, because we do not have abstractions for creating a database connection and `MATCH` query.
+You can create and start Kafka or Pulsar stream using GQLAlchemy. 
+<br>
 
-Now, rewrite the exact same query by using the functionality of GQLAlchemy's query builder:
+**Kafka stream** 
+```python
+from gqlalchemy import MemgraphPulsarStream
 
+stream = MemgraphPulsarStream(name="ratings_stream", topics=["ratings"], transform="movielens.rating", service_url="localhost:6650")
+db.create_stream(stream)
+db.start_stream(stream)
+```
+
+**Pulsar stream**
 ```python
-from gqlalchemy import match, Memgraph
+from gqlalchemy import MemgraphKafkaStream
+
+stream = MemgraphKafkaStream(name="ratings_stream", topics=["ratings"], transform="movielens.rating", bootstrap_servers="localhost:9093")
+db.create_stream(stream)
+db.start_stream(stream)
+```
+</details>
+
+<details>
+<summary>Import data from different sources 🗄️</summary>
```

Comment:
I would maybe add table to this title, to so people can see right away (before opening this section) that we can transform data from table to graph.

Reply:
Thought about what's the best way to do this, regarding the current dropdowns. I'll figure it out, but this is a great suggestion.